software_reseller/data/knowledge_article.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
+        <record id="software_reseller.knowledge_article_1_bdb37e60" model="knowledge.article" context="{'skip_check_member_access': True}">
+            <field name="sequence">1</field>
+            <field name="category">workspace</field>
+            <field name="last_edition_uid" ref="base.user_admin"/>
+            <field name="cover_image_id" ref="software_reseller.knowledge_cover_3_54a64d3e"/>
+            <field name="favorite_count">1</field>
```

Comment:
you can remove "favorite_count", as it has a default to 0, which is the correct value